corehq/apps/reports/standard/cases/utils.py

```diff
@@ -179,13 +183,24 @@ def query_location_restricted_forms(query, domain, couch_user):
     return query.filter(form_es.user_id(accessible_ids))
 
 
-def get_user_type(form_metadata, domain=None):
+def _get_system_form_types():
+    form_types = SYSTEM_FORM_XMLNS_MAP.copy()
+    form_types[DEDUPE_XMLNS] = gettext_lazy('Deduplication Rule')
+    form_types[XMLNS_DHIS2] = gettext_lazy('DHIS2 Integration')
```

Comment:
any reason not to add these to the `SYSTEM_FORM_XMLNS_MAP`

Reply:
The `SYSTEM_FORM_XMLNS_MAP` is being used in a few other functions to get form counts, so I was a little hesitant to alter it directly. Especially since adding in the extra mappings would exclude their form types from being counted in the previously mentioned functions. I'm not sure if this is an actual concern however, and if not then I could add them to `SYSTEM_FORM_XMLNS_MAP`.

Reply:
Looking again at the usages of `SYSTEM_FORM_XMLNS_MAP` I think it would make sense to add them to it.

Reply:
Agreed, this is consistent with the conclusions in
https://github.com/dimagi/commcare-hq/issues/31873
Love the change!